log/README.md

```diff
@@ -0,0 +1,159 @@
+# Contextual Logging
+
+## Log
+This library is a contextual logging library that makes use of context as part of the logging process. It is designed to make development easier by using the context variable to log instead of using a single global logger or passing a logger to every function.
+
+## Why use this library?
+### Do a lot of things in one operation
+The library focuses on doing multiple operations, whether it is adding more fields, logger, or logger configuration, in one chained operation. This makes setup and logging very simple, especially when they involve `Fields`.
+
+### Shallow Context Tree
+`Fields` are stored in the context tree when you use the `Onto` method. Doing many things in one operation allows you to produce a shallow context tree as you do not need to add `Fields` one-by-one. By finalizing the `Fields` operation using `Onto`, it ensures that it will only add all the provided `Fields` once. This is very beneficial when you have several fields to be logged and the bigger the system you have, it is highly likely that the number of fields will grow.
+
+### Greater control over `Fields` and `Logger`
+There are also many operations you can do on `Fields` as the library allows you to store fields in a variable for finer control over it. The `With` methods allows many different types of `Fields` to be entered and APIs like `Chain` and `Suppress` make `Fields` a lot more customizable.
```

Comment:
This is super nitpicky, but I think the last 2 words here,  "over it" is probably not needed? 
But your call, if you think the sentence flows better with them there

> There are also many operations you can do on `Fields` as the library allows you to store fields in a variable for finer control over it.

Reply:
Also, would we be able to make Chain and Suppress links to the relevant documentation for those APIs instead?

Reply:
I can link them to a godoc?